distributed/dashboard/tests/test_scheduler_bokeh.py

```diff
@@ -610,6 +611,41 @@ async def test_TaskGraph_order(c, s, a, b):
     assert gp.node_source.data["state"][gp.layout.index[y.key]] == "erred"
 
 
+####Adding TGroupGraph tests ######
+@gen_cluster(client=True)
+async def test_TGroupGraph(c, s, a, b):
```

Comment:
Added a test, but it's not fully working. 
After deleting the `futures` the source data doesn't get clear making the last assertion fail. Any clue what could be happening? 

Also both cases in this test result in only a single task group, meaning that the `arrow_source` data is always empty. Any ideas on how can I map/submit a computation to the client that gives us more than one task group so we can test for the `arrow_source` data?

Reply:
> After deleting the futures the source data doesn't get clear making the last assertion fail. Any clue what could be happening?

Responded with a suggestion below.  We probably need to wait until the scheduler learns of the change.  

> Also both cases in this test result in only a single task group, meaning that the arrow_source data is always empty. Any ideas on how can I map/submit a computation to the client that gives us more than one task group so we can test for the arrow_source data?

I think that TaskGroups currently ignore submit calls (otherwise we generate _a lot_ of task groups in some workloads).  I recommend using a dask array/dataframe workload instead with a few chunks/partitions.

Reply:
> I think that TaskGroups currently ignore submit calls (otherwise we generate a lot of task groups in some workloads).

I think we generate `TaskGroup`s anytime we add new tasks to the scheduler, including `submit`, `map`, etc. calls